sale_order_lot_selection/wizards/stock_lot_sale_order_wizard_line.py

```
@@ -0,0 +1,38 @@
+    )
+    lot_id = fields.Many2one(
+        "stock.lot",
+        string="Lot",
+        required=True,
+    )
+    product_id = fields.Many2one(related="lot_id.product_id")
+    quantity = fields.Float(string="Quantity")  # pylint: disable=W8113
+    max_qty = fields.Float(string="Max Quantity", related="lot_id.product_qty")
+
+    @api.constrains("quantity")
+    def _check_quantity(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_check_quantity`

Add return type annotation: `None`

(ANN202)